packages/embed/src/utils/get-transitive-search-params.ts

```diff
@@ -0,0 +1,14 @@
+export const getTransitiveSearchParams = (queryParams: Record<string, any>, transitiveSearchParams?: string[]) => {
+  const url = new URL(window.location.href)
+  const queryStringsWithSearchParams = { ...queryParams }
```

Comment:
This is not "query strings", there is just one "query string" in a URL. Maybe `queryParamsWithTransitiveParams`?